src/arco_era5/source_data.py

```diff
@@ -28,7 +28,7 @@
     "date-variable-pressure_level/{year}/{month:02d}/"
     "{day:02d}/{variable}/{pressure_level}.nc")
 
-STATIC_VARIABLES = (
+STATIC_VARIABLES = [
```

Comment:
Any reason why tuples didn't work?

Reply:
It's working with tuples,too. Reverted changes from list to tuple. Thanks for the suggestions.